app/src/main/java/com/example/harvestfresh/LoginActivity.java

```diff
@@ -21,15 +22,22 @@ public class LoginActivity extends AppCompatActivity {
     private EditText etUsername;
     private EditText etPassword;
```

Comment:
I believe Kal had a suggestion for renaming this that got missed from the previous PR. Will be good to address them here.

https://github.com/jjojan/HarvestFresh_Capstone/pull/7#discussion_r902960820